lib/session.js

```diff
@@ -1,9 +1,12 @@
-const
-  debug = require('debug')('telegraf:session-local'),
-  lowdb = require('lowdb'),
-  storageFileSync = require('lowdb/adapters/FileSync'),
-  storageFileAsync = require('lowdb/adapters/FileAsync'),
-  storageMemory = require('lowdb/adapters/Memory')
+const Debug = require('debug')
```

Comment:
How about to inline it, like so: `require('debug')('telegraf:session-local')`? Fewer lines in code is great :P

Reply:
inline will not work with ES6 modules which typescript uses

Also I personally do not expect anything different from imports in the first lines so I prefer more lines of code there. :)